chain-impl-mockchain/src/header/header.rs

```diff
@@ -237,26 +237,30 @@ impl Header {
         }
     }
 
-    pub fn to_content_eval_context(&self) -> HeaderContentEvalContext {
-        let gp_content = match self.block_version() {
+    pub fn to_consensus_eval_context(&self) -> ConsensusEvalContext {
```

Comment:
This `to_` naming bugs me a little: we don't convert the whole header to another thing, it's more like a `get_` method (or a simple getter, that would have no verb prefix per good Rust style, but this one may do some work to get the value).